docs(MIGRATION): add preset information for addon-backgrounds in 6.1 (#13180)
storybookjs/storybook: MIGRATION.md

## From version 6.0.x to 6.1.0

### Addon-backgrounds preset

> In 6.1 we there is an structural change in addon-backgrounds that ended up introducing an unintentional breaking change.

Reviewer: ```suggestion
In 6.1 we introduced an unintentional breaking change to `addon-backgrounds`.
```

> The addon needs decorators which come in a preset. The needed preset is ignored if you import the addon like this in `main.js`:

Reviewer: ```suggestion
The addon uses decorators which are set up automatically by a preset. The required preset is ignored if you register the addon in `main.js` withe the the `/register` entry point. This used to be valid in `v6.0.x` and earlier:
```

```js
module.exports = {
  stories: ['../**/*.stories.js'],
  addons: ['@storybook/addon-backgrounds/register'],
};
```

> To fix it, just replace `@storybook/addon-backgrounds/register` with `@storybook/addon-backgrounds`:

Reviewer: ```suggestion
To fix it, just replace `@storybook/addon-backgrounds/register` with `@storybook/addon-backgrounds`:

```
